src/formBuilder/defaults/shortAnswerInputs.js

```diff
@@ -176,6 +176,33 @@ function CardShortAnswerParameterInputs({
         }}
         className='card-modal-select'
       />
+      <h4>
```

Comment:
Lets factor this out into a common component that both shortAnswerInput and longAnswerInput is the same, to DRY out the code.